curriculum/challenges/english/01-responsive-web-design/basic-css/use-clockwise-notation-to-specify-the-margin-of-an-element.english.md

```diff
@@ -24,13 +24,15 @@ Use <code>Clockwise Notation</code> to give the element with the <code>blue-box<
 ```yml
 tests:
   - text: Your <code>blue-box</code> class should give the top of elements <code>40px</code> of <code>margin</code>.
-    testString: assert($(".blue-box").css("margin-top") === "40px", 'Your <code>blue-box</code> class should give the top of elements <code>40px</code> of <code>margin</code>.');
+    testString: assert($(".blue-box").css("margin-top") === "40px");
   - text: Your <code>blue-box</code> class should give the right of elements <code>20px</code> of <code>margin</code>.
-    testString: assert($(".blue-box").css("margin-right") === "20px", 'Your <code>blue-box</code> class should give the right of elements <code>20px</code> of <code>margin</code>.');
+    testString: assert($(".blue-box").css("margin-right") === "20px");
   - text: Your <code>blue-box</code> class should give the bottom of elements <code>20px</code> of <code>margin</code>.
-    testString: assert($(".blue-box").css("margin-bottom") === "20px", 'Your <code>blue-box</code> class should give the bottom of elements <code>20px</code> of <code>margin</code>.');
+    testString: assert($(".blue-box").css("margin-bottom") === "20px");
   - text: Your <code>blue-box</code> class should give the left of elements <code>40px</code> of <code>margin</code>.
-    testString: assert($(".blue-box").css("margin-left") === "40px", 'Your <code>blue-box</code> class should give the left of elements <code>40px</code> of <code>margin</code>.');
+    testString: assert($(".blue-box").css("margin-left") === "40px");
+  - text: You should use the clockwise notation to set the margin of <code>blue-box</code> class.
+    testString: const removeCssComments = str => str.replace(/<style>|<\/style>/g, '').replace(/\/\\*[\s\S]+?\*\//g, '');assert(/\.blue-box\s*{[\s\S]*margin:\s*\d+px\s+\d+px\s+\d+px\s+\d+px(;[\s\S]*}|\s*)/.test(removeCssComments(code)));
```

Comment:
In this case, it is better to use `$('style').text()` instead of `code`, then we get only a CSS code without a `<style>` tag.
And here is a little issue with  `/\/ -> \ <- \*[\s\S]+?\*\//g`.

Reply:
I forgot about using `$('style').text().  That is much better.  What kind of issues arise from the regex below?  I just want to make sure I create the best test as possible.  You must have some edge cases in mind.  Thanks.
```
.replace(/\/\\*[\s\S]+?\*\//g, '')
```

Reply:
the regex should match `/*` at the beginning, but it matches `/` one or more times. in your regexp - `\/\\*`, but should be `\/\*`.

Reply:
@ValeraS Got it.  I have made the two changes you suggested.  I got carried away with the escape character.  Thanks for reviewing.